test/tests/functional/pbs_default_qsub_argument.py

```diff
@@ -0,0 +1,87 @@
+# coding: utf-8
+
+# Copyright (C) 1994-2018 Altair Engineering, Inc.
+# For more information, contact Altair at www.altair.com.
+#
+# This file is part of the PBS Professional ("PBS Pro") software.
+#
+# Open Source License Information:
+#
+# PBS Pro is free software. You can redistribute it and/or modify it under the
+# terms of the GNU Affero General Public License as published by the Free
+# Software Foundation, either version 3 of the License, or (at your option) any
+# later version.
+#
+# PBS Pro is distributed in the hope that it will be useful, but WITHOUT ANY
+# WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+# FOR A PARTICULAR PURPOSE.
+# See the GNU Affero General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+# Commercial License Information:
+#
+# For a copy of the commercial license terms and conditions,
+# go to: (http://www.pbspro.com/UserArea/agreement.html)
+# or contact the Altair Legal Department.
+#
+# Altair’s dual-license business model allows companies, individuals, and
+# organizations to create proprietary derivative works of PBS Pro and
+# distribute them - whether embedded or bundled with other software -
+# under a commercial license agreement.
+#
+# Use of Altair’s trademarks, including but not limited to "PBS™",
+# "PBS Professional®", and "PBS Pro™" and Altair’s logos is subject to Altair's
+# trademark licensing policies.
+
+from tests.functional import *
+
+
+class TestDefaultQsubArgument(TestFunctional):
+    """
+    validate default_qsub_argument option.
+    """
+
+    def test_option_V(self):
+        """
+        This Test case exports an environment variable. It sets
+        default_qsub_argument with -V option and then submits a job.
+        It expects the Variable list to contain the exported environment
+        variable.
+        """
+        os.environ["SET_IN_SUBMISSION"] = "true"
+        self.server.manager(MGR_CMD_SET, SERVER,
+                            {'default_qsub_arguments': '-V'})
+        j = Job(ROOT_USER)
+        jid = self.server.submit(j)
+        env_var = os.environ.get("SET_IN_SUBMISSION")
+        self.server.expect(JOB, {'Variable_List': (MATCH_RE,
+                           'SET_IN_SUBMISSION=%s' % env_var)},
+                           id=jid)
+
+    def test_option_V_with_script(self):
+        """
+        This Test case exports an environment variable. It sets
+        -V option through a job script directive and then changes
+        the value of the environment variable. It expects another job
+        script to have the changed value of the environment varible.
```

Comment:
Instead of saying another job script we can say that "The job under execution should get the updated value of the environment variable" since its(Job Script Directives) precedence is higher than the others"